Makefile

```diff
@@ -163,7 +171,10 @@ e2e-prerequisites: $(GINKGO) e2e-templates e2e-image test-e2e-image-prerequisite
 E2E_GINKGO_ARGS ?=
 .PHONY: test-e2e ## Run e2e tests using clusterctl
 test-e2e: e2e-prerequisites ## Run e2e tests
-	time $(GINKGO) --failFast -trace -progress -v -tags=e2e --nodes=$(E2E_GINKGO_PARALLEL) $(E2E_GINKGO_ARGS) ./test/e2e/suites/e2e/... -- -config-path="$(E2E_CONF_PATH)" -artifacts-folder="$(ARTIFACTS)" --data-folder="$(E2E_DATA_DIR)" $(E2E_ARGS)
+	time $(GINKGO) --failFast -trace -progress -v -tags=e2e --nodes=$(E2E_GINKGO_PARALLEL) \
+			--focus="$(E2E_GINKGO_FOCUS)" $(_SKIP_ARGS) $(E2E_GINKGO_ARGS) ./test/e2e/suites/e2e/... -- \
```

Comment:
Does this error if `E2E_GINKGO_FOCUS` is empty? Do we need the same dance as on lines 131-133 above for skip?

If not, can we also omit the dance for `E2E_GINKGO_SKIP`?

Reply:
It works even if `E2E_GINKGO_FOCUS` is empty. I actually stole it from [CAPI](https://github.com/kubernetes-sigs/cluster-api/blob/main/Makefile#L777-L778). They also have the same `_SKIP_ARGS` and I think this is because an empty string would match everything. So empty focus is no problem, we just run all the tests. Skipping everything on the other hand would not be great.